### PR TITLE
Regulations site, new snippets, improvements of grids

### PR DESCRIPTION
**Pull request by KamilWo**
Latest version of code of the FPBZ Theme.

### CODE REVIEW

File: addons/bestja_frontend_fpbz/static/src/img/content/down_arrow.svg [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		 M77.53,57.468c-1.989,4.632-4.629,8.684-8.016,12.047c-3.363,3.34-7.406,6-12.047,8.015c-4.641,2.016-9.516,3-14.625,3
+		c-6.797,0-13.078-1.687-18.844-5.062c-5.766-3.375-10.369-7.963-13.734-13.734C6.88,55.927,5.203,49.593,5.203,42.796
+		s1.687-13.078,5.062-18.843C13.64,18.187,18.234,13.594,24,10.219c5.766-3.375,12.047-5.063,18.844-5.063
+		c5.109,0,9.984,0.984,14.625,3c4.641,2.016,8.66,4.699,12.047,8.062c3.363,3.34,6.027,7.368,8.016,12c1.996,4.649,3,9.469,3,14.578
+		C80.53,47.906,79.526,52.819,77.53,57.468z"/>
+	<path fill="#F29223" d="M64.03,32.437L43.124,53.249L21.703,32.343c-1.022-0.998-2.672-0.984-3.75,0.094c-0.938,0.938-0.93,2.659,0.094,3.656
+		L41.39,58.827c0.95,0.925,2.648,1.013,3.656,0l22.64-22.734c0.515-0.516,0.75-1.125,0.75-1.828c0-0.797-0.234-1.406-0.75-1.828
+		C66.749,31.359,65.111,31.361,64.03,32.437z"/>
+</g>
+<g>
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/static/src/img/content/down_arrow.svg]: Empty `<g>` tags.

File: addons/bestja_frontend_fpbz/templates/pages/home_page.xml [hunk 2]
```diff
@@ -1,398 +1,400 @@
...
+                    <section class="oe_snippet_body">
+                        <div class="container-fluid quote_div">
+                            <div class="row">
+                                <div class="col-md-offset-1 col-md-10">
+                                    <blockquote>Wolontariat daje mi 
+                                        przede wszystkim satysfakcję z pomagania, 
+                                        a także możliwość kreatywnego działania. Poznaję nowych ludzi, 
+                                        uczę się pracy w zespole i rozwijam swoje zainteresowania.
+                                    </blockquote>
+                                    <span class="podpis_pod_cytatem">
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/templates/pages/home_page.xml]: This is a regression, the class `podpis_pod_cytatem` has already been replaced by its English version.

File: addons/bestja_frontend_fpbz/templates/pages/footer.xml [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+                                    <li class="mt8 mb8">
+                                        <a href="http://www.wolontariat.bankizywnosci.pl/">O nas</a>
+                                    </li>
+                                </ul>                                
+                            </div>
+                            <div class="col-md-2 centered text-center" />
+                            <div class="col-md-5 mt16 footer_icons_container">
+                                <ul class="col-md-12 footer_icons">
+                                    <li>
+                                        <a class="facebook" href="https://www.facebook.com/bankizywnosci?ref=ts"></a>
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/templates/pages/footer.xml]: We can load it from the database. The same applies to Youtube and Twitter URL – these are saved in Odoo settings.

File: addons/bestja_frontend_fpbz/templates/pages/regulations.xml [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/templates/pages/regulations.xml]: This should be put in the `bestja_page_fixtures_fpbz` module. (If it does not exist, it should be created).

**Comment by KrzysiekJ** [comment]
There is no reasonable background color for `.main_top_photo_text`, so when background image is not loaded, the text is hardly readable.

File: addons/bestja_frontend_fpbz/static/src/css/custom.css [hunk 5]
```diff
@@ -1,563 +1,677 @@
...
+        margin-right: 0;
+    }
+}
+
+section .top-section-container{
+    margin-left: 0;
+    margin-right: 0;
+}
+
+h1, .h1{
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/static/src/css/custom.css]: Is repeating tag names in class names necessary? Do we actually use those classes?

File: addons/bestja_frontend_fpbz/static/src/img/content/down_arrow.svg (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/static/src/img/content/down_arrow.svg]: Empty tags have been removed.

File: addons/bestja_frontend_fpbz/templates/pages/home_page.xml (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/templates/pages/home_page.xml]: Ok, thank you, I fixed this.

File: addons/bestja_frontend_fpbz/templates/pages/footer.xml (hunk 3 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/templates/pages/footer.xml]: Of course, thank you, change has been made.

File: addons/bestja_frontend_fpbz/templates/pages/regulations.xml (hunk 4 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/templates/pages/regulations.xml]: Module has been created, pages have been moved.

File: addons/bestja_frontend_fpbz/static/src/css/custom.css (hunk 5 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/static/src/css/custom.css]: That is how Boostrap defines its css for headers. Should I remove those classes?

**Comment by KamilWo** [comment]
I have changed background for ``` .main_top_photo_text ``` to ```#777777```.

File: addons/bestja_frontend_fpbz/templates/snippets/get_knowledge.xml [hunk 6]
```diff
@@ -17,30 +17,30 @@
                                 </div>
                                 <div class="col-md-10 centered">
                                     <div class="col-md-3 centered">
-                                        <div class="media_iframe_video" data-src="//www.youtube.com/embed/AuqxJgJTwaI?autoplay=0" style="">
+                                        <div class="media_iframe_video" data-src="http://www.youtube.com/embed/AuqxJgJTwaI?autoplay=0" style="">
```
Comment (KrzysiekJ) [inline addons/bestja_frontend_fpbz/templates/snippets/get_knowledge.xml]: These `http` URLs to Youtube embed codes return 301 responses with redirects to `https` version, so we should put `https` directly here.

File: addons/bestja_page_fixtures_fpbz/templates/pages/home_page.xml [hunk 7]
```diff
@@ -5,85 +5,91 @@
...
-                                    <span class="direction_down color_orange">&lt;</span>
+                                <div id="down_arrow_scroll" class="text-center mt64">
+                                    <img class="img arrow_img" src="/bestja_frontend_fpbz/static/src/img/content/down_arrow.svg"/>
                                 </div>
                             </div>
                         </div>
+                        <script>
+                            // Background picture will fit viewport
+                            var height = window.innerHeight - document.getElementsByClassName('navbar-static-top')[0].offsetHeight - 30;
+                            if(window.innerHeight > 1920){
```
Comment (KrzysiekJ) [inline addons/bestja_page_fixtures_fpbz/templates/pages/home_page.xml]: Why not 1080?

File: addons/bestja_frontend_fpbz/templates/snippets/get_knowledge.xml (hunk 6 above)
Comment (KamilWo) [inline addons/bestja_frontend_fpbz/templates/snippets/get_knowledge.xml]: divs with media_iframe_video should not be there at all. They will be removed.

File: addons/bestja_page_fixtures_fpbz/templates/pages/home_page.xml (hunk 7 above)
Comment (KamilWo) [inline addons/bestja_page_fixtures_fpbz/templates/pages/home_page.xml]: True, there should be 1080. Thank you.

**Comment by KrzysiekJ** [comment]
Merged in 424a318.

**Pull request closed without merging**